src/Initializer/Parameters/ModelParameters.cpp

```diff
@@ -34,6 +34,29 @@ ITMParameters readITMParameters(ParameterReader* baseReader) {
       itmEnabled, itmStartingTime, itmDuration, itmVelocityScalingFactor, reflectionType};
 }
 
+DamagedElasticParameters readDamagedElasticParameters(ParameterReader* baseReader){
+    auto* reader = baseReader -> readSubNode("equations");
+    const auto epsInitxx = reader->readWithDefault<real>("epsinitxx", 3.7986e-4);
+    const auto epsInityy = reader->readWithDefault<real>("epsinityy", -1.0383e-3);
+    const auto epsInitzz = reader->readWithDefault<real>("epsinitzz", -1.0072e-3);
+    const auto epsInitxy = reader->readWithDefault<real>("epsinitxy", 1.0909e-3);
```

Comment:
Should we really keep these values as default?!

Reply:
@Aangniu should be able to comment better, I guess. Are these values picked for some particular scenario? What would be a good artificial benchmarking scenario with generic intuitive initial values?

Reply:
These values are chosen according to the tpv5 benchmark. The differences from the elastic case is here we need to incorporate those applied initial stress on the fault also in the material. So these initial strain will lead to 120 MPa normal traction and 70 MPa shear traction.

Reply:
We should consider 0 as a default then.

Reply:
Modified